docs/User/SWCI_APIs.md

```diff
@@ -6,14 +6,14 @@ The following API methods help to invoke SWCI operations programmatically. It is
 NOTE:Arguments that do not have default values are mandatory.
 
 </blockquote>
-The python3 program needs to import the SWCI class, and then use the below APIs.
+The python3 program needs to import the SWCI class first, and then use the below APIs.
```

Comment:
There is one more note that is missing. Look at page 26 of User guide.

Reply:
Resolved